src/libaktualizr/storage/storage_common_test.cc

```diff
@@ -494,6 +494,47 @@ TEST(storage, store_target) {
   }
 }
 
+TEST(storage, list_remove_targets) {
```

Comment:
I'd like a brief comment explaining the precise features you're trying to test.

(I've given up on maintaining the actions list these days, since no one seems to care about it, but I still think the comments for each test that clarify the important points of the test are very useful.)

Reply:
Yes that's too bad actually. Part of the problem might be that it's a bit hard to keep track as it needs to be changed in two places.

If we could just define them where they are tested and have an automated tool to collect them, it would be a bit more helpful. But then, it would maybe take some non-trivial effort.

Reply:
> If we could just define them where they are tested and have an automated tool to collect them, it would be a bit more helpful. But then, it would maybe take some non-trivial effort.

100% agreed, and it was my hope that we would eventually do that. The one problem I foresaw was how to structure the information, though. That was the nice thing about the list. In any case, it probably isn't hard to read the comments around the tests; we started doing something similar with zephyr integration, but that also got stalled.